addons/l10n_cn_city/__manifest__.py

```diff
@@ -4,7 +4,6 @@
 
 {
     'name': 'China - City Data',
-    'countries': ['cn'],
```

Comment:
@odoo/rd-accounting, does anyone know why this module is not auto-installed?
Is it a mistake or have some reason?

Reply:
It is probably only to avoid creating a lot of data if the users don't want it.
Note that it will also set `enforce_cities=True`, which might be undesired

Reply:
Basically this: https://github.com/odoo/odoo/commit/c3f0915b2b49d100257a1ac0f20d56beb3307d0d

Reply:
okay, got it.